### PR TITLE
#167335779 The operator should be able to set the working environment 

### PR DESCRIPTION
**Pull request by Ally4**
#### What does this PR do?
This PR is for the setting of the working environment for challenge 3
#### Description of Task to be completed?
Creation of develop2 as a branch from master
Pushing develop2 on the Github
Make a pull request on develop2 from develop
Merging develop to develop2
#### How should this be manually tested?
It can be tested by going on the local repository
And check into the branches 
Then select develop2
#### Any background context you want to provide?
No,
#### What are the relevant pivotal tracker stories?
#167335779

### CODE REVIEW

**Comment by b0nbon1** [comment]
Why are there html files commits too??
and you have not worked on eslint still

File: UI/html/update.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+        <meta name="viewport" content="width= device-width, initial-scale=1.0">
+    <title>update</title>
+    <link rel="stylesheet" href="../css/main.css">
```
Comment (Kasulejoseph) [inline UI/html/update.html:7]: Why do you work on the HTML files in a backend PR? you should raise a different PR if you want to make changes on the frontend.

File: coverage/index.html [hunk 2]
```diff
@@ -0,0 +1,164 @@
+<!doctype html>
```
Comment (Kasulejoseph) [inline coverage/index.html:1]: Coverage folder should be ignored from the .gitignore file, don't push coverage folder.

File: server/controllers/sold.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+/* eslint-disable comma-spacing */
+/* eslint-disable no-irregular-whitespace */
+/* eslint-disable no-trailing-spaces */
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable indent */
+/* eslint-disable linebreak-style */
+/* eslint-disable import/no-unresolved */
+// import moment from 'moment';
+// import validateUpdateStatus from '../validators/posts';
+/* eslint-disable padded-blocks */
```
Comment (Kasulejoseph) [inline server/controllers/sold.js:14]: Configure eslint, use an eslint file.

File: server/models/posts.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+/* eslint-disable linebreak-style */
+const posts = [{
+    id: 1,
+    owner:'sunny1@gmail.com',
+    status:'Available',
+    type:'2room',
+    state:'jhjhhgh',
+    city:'Kigali',
```
Comment (Kasulejoseph) [inline server/models/posts.js:8]: Remove this hardcoded data.

**Pull request merged**